README.md

```diff
@@ -59,24 +59,52 @@ When triggered manually it will _not_ push the images to the registry.
 
 Many products apply Stackable-specific patches, managed by [Patchable](rust/patchable).
 
+Patchable doesn't _edit_ anything by itself. Instead, it's a uniform way to apply a set of patches
+to an upstream Git repository, and then export your local changes back into patch files.
+You can then edit the branch created by patchable using any Git frontend, such as the git CLI or
```

Comment:
This sounds like you apply and export the local changes before editing with git et.al .... 

Maybe something like

> Instead, it's a uniform way to apply a set of patches to an upstream Git repository, and then export your local changes back into patch files.
> It doesn't care how you make the local changes before exporting to patchfiles again - you can edit the branch created by patchable using any Git frontend, such as the git CLI or ...

Reply:
1366a0c4f1d8de97a3bc8714d9c77c78aba03ab5

Reply:
Sorry for being nitpicky here, but could you remove the 'then' from the sentence? That was what mostly tripped me up and made me think "apply changes, export, then locally edit" which had me heading of in a wrong direction mentally :)

Reply:
908774478c16a361a8e6e903200061df40cdbb8a